eng/pipelines/templates/stages/powershell-tests.yml

```diff
@@ -0,0 +1,20 @@
+parameters:
```

Comment:
I would suggest following the same pattern as .NET https://github.com/Azure/azure-sdk-for-net/blob/main/eng/scripts/ci.yml

Reply:
Thanks for the suggestion, I just deleted this file. 

I checked the `run-pester-tests.yml` [file](https://github.com/Azure/azure-sdk-for-net/blob/main/eng/common/pipelines/templates/steps/run-pester-tests.yml) shared by @raych1 in this [comment](https://github.com/Azure/azure-sdk-for-java/pull/47079#issuecomment-3522861075)

Seems the .NET [ci.yml](https://github.com/Azure/azure-sdk-for-net/blob/main/eng/scripts/ci.yml) you shared is not using `run-pester-tests.yml` which is shared by @raych1 ? And I did not find the similar `ci.ym`l file in similar location in Java repo: https://github.com/Azure/azure-sdk-for-java/tree/main/eng/scripts

I may not have much context on CI and pipeline. Could I get more advices on what code change is needed for Java to run the pester tests in CI?

Reply:
follow up issue created: https://github.com/Azure/azure-sdk-for-java/issues/47398
Thanks for the suggestion!